lib/node/nodes/aggregate-intersection.js

```diff
@@ -77,3 +77,16 @@ var queryAggregateTemplate = Node.template([
     'WHERE ST_Intersects(_cdb_analysis_source.the_geom, _cdb_analysis_target.the_geom)',
     'GROUP BY {{=it.groupByColumns}}'
 ].join('\n'));
+
+AggregateIntersection.prototype.computeRequirements = function(databaseService, limits, callback) {
+    // we estimate the maximum possible number of rows of the result
+    var product = this.source.estimatedRequirements.numberOfRows *
```

Comment:
We have several places where we don't follow `getter/setter` pattern. IMHO in Javascript is not useful, you can always get a member property directly with `.`. My advice is to avoid the `Cannot read property 'wadus' of null` using default values and so on.